### PR TITLE
Findone test1

### PR DESCRIPTION
**Pull request by justmic007**
Test both happy and sad for findById and updateById on User endpoint for crud-api

### CODE REVIEW

File: src/persistence/mongoose/User/queries.test.js [hunk 1]
```diff
@@ -42,4 +42,51 @@ describe('Users queries', () => {
...
   })
+
+  describe('findById', () => {
+    it('can find a single user', async () => {
+      const user = await factory.create('User');
+
+      const findUser = await userQueries.findById({ payload: { uuid: user.uuid } });
+
+      expect(findUser).toBeDefined();
+      expect(findUser).toHaveProperty('uuid'); // eslint-disable-line
```
Comment (Ehbraheem) [inline src/persistence/mongoose/User/queries.test.js]: Remove `eslint-disable comment

File: src/persistence/mongoose/User/queries.test.js [hunk 2]
```diff
@@ -42,4 +42,51 @@ describe('Users queries', () => {
...
+  describe('updateById', () => {
+    it('update a single user', async () => {
+      const user = await factory.create('User');
+      const updateUser = await userQueries.updateById({ payload: user.uuid });
+
+      expect(updateUser).toBeDefined()
+      expect(updateUser).toHaveProperty('nModified');
+      expect(updateUser).toHaveProperty('n');
+      expect(updateUser).toHaveProperty('ok');
+      expect(updateUser).toHaveProperty('ok');
```
Comment (Ehbraheem) [inline src/persistence/mongoose/User/queries.test.js]: Why duplicate?

File: src/persistence/mongoose/User/queries.test.js [hunk 3]
```diff
@@ -42,4 +42,51 @@ describe('Users queries', () => {
...
+      expect(updateUser['nModified']).toEqual(1);
+      expect(updateUser['n']).toEqual(1);
```
Comment (Ehbraheem) [inline src/persistence/mongoose/User/queries.test.js]: I'd suggest you do an extra findOne to get the updated document.

File: test/utils/databaseCleaner.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
-import mongoose from 'mongoose';
-import Cleaner from 'database-cleaner';
+// import mongoose from 'mongoose';
+// import Cleaner from 'database-cleaner';
 
-var dbCleaner = new Cleaner('mongodb');
+// var dbCleaner = new Cleaner('mongodb');
```
Comment (Ehbraheem) [inline test/utils/databaseCleaner.js]: Why did you comment out these lines

Comment (justmic007) [reply test/utils/databaseCleaner.js]: oh!

**Pull request merged**